routing/cross_mwm_connector.cpp

```diff
@@ -1,173 +1,18 @@
-  // to be a transition segment. If not, |segment| is not a transition segment.
-  // Please see documentation on CrossMwmGraph::IsTransition() method for details.
-  bool const isEnter = (segment.IsForward() == transition.m_forwardIsEnter);
-  return isEnter != isOutgoing;
-}
-
-Segment const * CrossMwmConnector::GetTransition(uint64_t osmId, uint32_t segmentIdx,
-                                                 bool isEnter) const
-{
-  auto fIt = m_osmIdToFeatureId.find(osmId);
-  if (fIt == m_osmIdToFeatureId.cend())
-    return nullptr;
-
-  uint32_t const featureId = fIt->second;
-
-  auto tIt = m_transitions.find(Key(featureId, segmentIdx));
-  if (tIt == m_transitions.cend())
-    return nullptr;
-
-  Transition const & transition = tIt->second;
-  CHECK_EQUAL(transition.m_osmId, osmId,
-              ("feature:", featureId, ", segment:", segmentIdx, ", point:",
-               MercatorBounds::ToLatLon(transition.m_frontPoint)));
-  bool const isForward = transition.m_forwardIsEnter == isEnter;
-  if (transition.m_oneWay && !isForward)
-    return nullptr;
-
-  Segment const & segment =
-      isEnter ? GetEnter(transition.m_enterIdx) : GetExit(transition.m_exitIdx);
-  CHECK_EQUAL(segment.IsForward(), isForward, ("osmId:", osmId, ", segment:", segment, ", point:",
-                                               MercatorBounds::ToLatLon(transition.m_frontPoint)));
-  return &segment;
-}
-
-m2::PointD const & CrossMwmConnector::GetPoint(Segment const & segment, bool front) const
-{
-  Transition const & transition = GetTransition(segment);
-  return segment.IsForward() == front ? transition.m_frontPoint : transition.m_backPoint;
-}
-
-void CrossMwmConnector::GetEdgeList(Segment const & segment, bool isOutgoing,
-                                    std::vector<SegmentEdge> & edges) const
+namespace connector
 {
-  Transition const & transition = GetTransition(segment);
-  if (isOutgoing)
-  {
-    ASSERT_NOT_EQUAL(transition.m_enterIdx, kFakeId, ());
-    for (size_t exitIdx = 0; exitIdx < m_exits.size(); ++exitIdx)
-    {
-      Weight const weight = GetWeight(base::asserted_cast<size_t>(transition.m_enterIdx), exitIdx);
-      AddEdge(m_exits[exitIdx], weight, edges);
-    }
-  }
-  else
-  {
-    ASSERT_NOT_EQUAL(transition.m_exitIdx, kFakeId, ());
-    for (size_t enterIdx = 0; enterIdx < m_enters.size(); ++enterIdx)
-    {
-      Weight const weight = GetWeight(enterIdx, base::asserted_cast<size_t>(transition.m_exitIdx));
-      AddEdge(m_enters[enterIdx], weight, edges);
-    }
-  }
-}
-
-bool CrossMwmConnector::WeightsWereLoaded() const
-{
-  switch (m_weightsLoadState)
-  {
-  case WeightsLoadState::Unknown:
-  case WeightsLoadState::ReadyToLoad: return false;
-  case WeightsLoadState::NotExists:
-  case WeightsLoadState::Loaded: return true;
-  }
-}
-
-std::string DebugPrint(CrossMwmConnector::WeightsLoadState state)
+std::string DebugPrint(WeightsLoadState state)
```

Comment:
`using namespace std;` && s/std::string/string ?

Reply:
Let's leave it as is. There's only one place where std:: is used in this file.

Reply:
ok